_posts/2018-12-10-blog-lets-create some-glip-bots-with-python.markdown

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title:  "Let's create some glip bots with python"
+date:   2018-12-10 13:31:07 +0800
+categories: blog
+---
+
+![ ](https://github.com/zxdong262/ringcentral-chatbot-factory-py/blob/master/screenshots/wanted.jpg?raw=true)
+
+This is a intro blog about my recent work: [ringcentral-chatbot-python](https://github.com/zxdong262/ringcentral-chatbot-python).
+
+The project is part of our chatbot plan: Make writing a glip chatbot easy and fun. It is not mature enough for public yet, still need more documents and adding more features.
+
+It do works and is really easy to work with, you do not even need to know any python to get it running and talking. So ringcentral developers, let's create some chatbot with python, and give some feedback.
```

Comment:
"It does work ..."
"Do" is the verb here, and you need to change to "does" for third person perspective. Verbs that come after "do" or "does" need to be in first person present tense, like 'I do care about quality', 'you did use the keyboard', or 'he does eat chocolates'.